plugin/plugin.go

```diff
@@ -54,6 +54,13 @@ func run(dockerClient *docker.Client, weave *weaveapi.Client, address, meshAddre
 		defer os.Remove(meshAddress)
 		defer meshListener.Close()
 	}
+	if !isPluginV2 {
```

Comment:
Ouch, too late, but this could go to `if meshAddress != "" {` branch, as the network should be created only if the local scope driver (`weavemesh`) is started.